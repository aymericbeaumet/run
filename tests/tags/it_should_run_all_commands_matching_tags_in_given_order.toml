tags = ["c", "b", "a"]

[[run]]
tags = ["a", "b"]
cmd = ["bash", "-c", "echo a"]

[[run]]
tags = ["b", "c"]
cmd = ["bash", "-c", "echo b"]

[[run]]
tags = ["c"]
cmd = ["bash", "-c", "echo c"]