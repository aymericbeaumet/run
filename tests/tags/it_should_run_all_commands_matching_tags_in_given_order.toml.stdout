[bash] c
[bash] b
[bash] a
